openapi/openapi.yaml

```diff
@@ -629,6 +629,19 @@ components:
           description: The type (or types) of descriptors available.
           items:
             $ref: "#/components/schemas/DescriptorType"
+        descriptor_type_version:
+          type: object
```

Comment:
Should this be of `type` array?
My understanding is that CWL workflows can have a mixture of tools of different cwl versions.
https://github.com/common-workflow-language/cwltool/issues/1321

Reply:
@denis-yuen that makes sense. I will switch this over to a an array on Monday.

One thing I did consider, was changing the existing `descriptor_type` to contain this information since it is essentially being duplicated. But I figured that would be a harder change to get into the spec then a new field